modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+[id="persistent-storage-csi-vsphere-install-issues_{context}"]
+= Removing a non-Red Hat vSphere CSI Operator Driver
+
+If you have installed a non-Red Hat vSphere Container Storage Interface (CSI) Operator Driver, updates for your cluster to the next major version of {product-title}, such as 4.11, may be disabled due to {product-title} 4.10 having a built-in version of the vSphere CSI Operator Driver.
+
+{product-title} 4.10 clusters are still fully supported, and updates to bug fix versions of 4.10, such as 4.10.z, are not blocked, but you must correct this state by removing the non-Red Hat vSphere CSI Operator Driver before updates to next major version of {product-title} can occur. Removing the non-Red Hat vSphere CSI driver does not require deletion of associated persistent volume (PV) objects, and no data loss should occur.
+
```

Comment:
same here for "CSI Operator Driver" here

Reply:
Implemented this comment.